wordpress-mobile/WordPress-Android
# Create media picker launcher

Create a media picker launcher that is used to trigger either a PhotoPickerActivity or MediaPickerActivity based on the flag. This should be replacing all the places where we start a PhotoPickerActivity. As part of this PR I've extracted the constants into a reused class called `MediaPickerConstants` so minimize a chance of breaking the contract when switching the flags.

To test:
- Turn off the flag
- Make sure the photo picker works (Image/Video/Gallery blocks, Stories) 
- Turn on the flag
- Make sure the photo picker works (Image/Video/Gallery blocks, Stories)

PR submission checklist:
